libscan/http/scanreporthandler.go

```diff
@@ -29,7 +28,6 @@ func ScanReport(lib libscan.Libscan) h.HandlerFunc {
 		sr, ok, err := lib.ScanReport(ctx, hash)
 		if err != nil {
 			h.Error(w, fmt.Sprintf("error receiving scanreport"), h.StatusInternalServerError)
-			log.Printf("failed to retrieve scanreport for %v: %v", hash, err)
```

Comment:
I feel these logs are still convenient - the error above it is returned to the client, we would not see those errors on our log stream watching stdout server side

Reply:
this message echos for all log statements removed in http handler